src/main/java/uk/gov/justice/digital/hmpps/keyworker/services/QueueAdminService.kt

```diff
@@ -12,12 +12,12 @@ import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression
 import org.springframework.stereotype.Service
 
 @Service
-@ConditionalOnExpression("{'aws', 'localstack'}.contains('\${sqs.provider}')")
+@ConditionalOnExpression("{'aws', 'localstack'}.contains('\${offender-events-sqs.provider}')")
 class QueueAdminService(
```

Comment:
This will need to change a bit more in future PRs as it's still generic. Maybe we could extend it to so that it takes a queue name as a path variable. Or we have two separate admin queue services.